packages/cli/src/index.ts

```diff
@@ -587,20 +587,93 @@ const main = async () => {
     }
   }
 
-  if (!targetCommand) {
-    const sub = param(subcommand);
-    console.error(error(`The ${sub} subcommand does not exist`));
-    return 1;
-  }
-
   const metric = metrics();
   let exitCode;
   const eventCategory = 'Exit Code';
 
   try {
     const start = Date.now();
-    const full = require(`./commands/${targetCommand}`).default;
-    exitCode = await full(client);
+    let func: any;
```

Comment:
```suggestion
    let func: (c: Client) => Promise<number>;
```

Reply:
This was failing for me which is why I left it as `any`.

I'll try again in another PR and see if we can get this more strict.

Another option is to move the imports to the top because they're already bundled into a single file via ncc.